Dalamud/Logging/PluginLog.cs

```diff
@@ -8,6 +8,131 @@ namespace Dalamud.Logging
     /// </summary>
     public static class PluginLog
     {
+        #region "Log" prefixed Serilog style methods
```

Comment:
When recompiling plugins after removing this earlier, these were used quite a lot. So back in they go.